### ISSUE TITLE
Config file example

### ISSUE THREAD

**Issue by kaiquewdev** [llSourcell/AI_Composer#3]
Can you share an configuration file sample?

**Comment by sshh12** [comment]
An example is in rnn.py

**Comment by beeyeas** [comment]
rnn.py (https://github.com/llSourcell/AI_Composer/blob/master/rnn.py) has DefaultConfig, it helps to see what the --config-file would like, i tried like this but no luck


Num Layers=2
Hidden Size=200
Melody Coeff=0.5
Dropout Prob=0.5
Input Dropout Prob=0.8
Cell Type='lstm'
Time Batch Len=128
Learning Rate=5e-3
Decay=0.9

**Comment by iantheconway** [comment]
It looks like in rnn_sample.py line 30 the module is expecting that the new_config_file.config file be a cPickle that it can load. In rnn.py there is a class which has those parameters as members, I suspect that a pickle containing an instance of that class is saved during training. I looked in my model folder and I found a file which has a .config extension: nl_2_hs_200_mc_0p5_dp_0p5_idp_0p8_tb_128.config
instead of calling python rnn_sample.py --config_file new_config_file.config I called python rnn_sample.py --config_file ./models/0322_1020/nl_2_hs_200_mc_0p5_dp_0p5_idp_0p8_tb_128.config
This seems to have got me past that error, but no I'm crashing on something related to midi.py.

**Comment by Aashit-Sharma** [comment]
@iantheconway When I run the rnn_sample , it shows me

> usage: rnn_sample.py [-h] --config_file CONFIG_FILE [--sample_melody]
                     [--sample_harmony] [--sample_seq {random,chords}]
                     [--conditioning CONDITIONING]
                     [--sample_length SAMPLE_LENGTH]

Script to generated a MIDI file sample from a trained model.

optional arguments:
  -h, --help            show this help message and exit
  --config_file CONFIG_FILE
  --sample_melody
  --sample_harmony
  --sample_seq {random,chords}
  --conditioning CONDITIONING
  --sample_length SAMPLE_LENGTH


The problem is that I cannot find any generated MIDI file. It shows no error and yet no output.
Did you also get this ?
